man/docker-run.1.md

```diff
@@ -417,8 +417,13 @@ standard input.
     Ulimit options
 
 **-v**, **--volume**=[]
-   Bind mount a volume (e.g., from the host: -v /host:/container, from Docker: -v /container)
-
+   Create a bind mount (format: `[host-dir:]container-dir[:rw|ro]`)
+   (e.g., using -v /host-dir:/container-dir, bind mounts /host-dir in the
+host to /container-dir in the Docker container)
+   
+   If 'host-dir' is missing, then docker automatically creates the new volume
+on the host.
```

Comment:
Yes, and good idea but it would be the only place where "Docker host" is mentioned in the document.  Host without the adjective is used throughout.  So I think it's better to keep with host in this case.